diff --git a/build/lib/infiniteQueryBehavior.mjs b/build/lib/infiniteQueryBehavior.mjs
index 34fde2ec178d7af4af39ae825a3a79ccded05000..b0d46777ccba5353c768c16882ae8f1547ff37c8 100644
--- a/build/lib/infiniteQueryBehavior.mjs
+++ b/build/lib/infiniteQueryBehavior.mjs
@@ -103,7 +103,13 @@ function infiniteQueryBehavior() {
         const finalPromise = promise.then(pages => ({
           pages,
           pageParams: newPageParams
-        }));
+        })).then((data) => {
+          if (typeof context.options.meta?.interceptInfiniteQueryBehavior === "function") {
+            return context.options.meta.interceptInfiniteQueryBehavior(data);
+          } else {
+            return data;
+          }
+        });
         return finalPromise;
       };
     }